### ISSUE TITLE
Vimeo links via API

### ISSUE THREAD

**Issue by cscheid** [ieee-vgtc/ieeevis.org#366]
@codementum found out that Vimeo has an API: https://developer.vimeo.com/api/start

Can we use it?

**Comment by yelper** [comment]
I'll look into this tonight (shooting for a csv file for compatibility purposes).

**Comment by yelper** [comment]
Here's a sample: [2017-videos.json](https://alper.datav.is/assets/tmp/2017-videos.json).  The python script is simple, but has some secrets in it (that I'm happy to share, if we have a safe place for them).

**Comment by cscheid** [comment]
> On Nov 3, 2017, at 12:20 AM, Alper Sarikaya <notifications@github.com> wrote:
> 
> Here's a sample: 2017-videos.json. The python script is simple, but has some secrets in it (that I'm happy to share, if we have a safe place for them).

Awesome!

Agreed that we need to figure out a place to put shared secrets. We don’t have a good solution right now.

Ideas?

**Comment by yelper** [comment]
One idea could be to use Keybase's GitHub implementation (but requires people set up a Keybase account): <https://keybase.io/blog/encrypted-git-for-everyone>.  

A benefit of Keybase is that they have ["teams"](https://keybase.io/blog/introducing-keybase-teams), which would allow us to have an [encrypted working space](https://keybase.io/docs/kbfs) (it works just like Dropbox, but only pulls down file contents when the contents are requested, otherwise it just puts dummy files there).

Another option is to encrypt secrets in this repo, using something like [Blackbox](https://github.com/StackExchange/blackbox) to encrypt files in-place, but this requires that people/deployment have PGP keypairs on hand (kind of annoying, and the problem Keybase was built to solve).  [git-crypt](https://www.agwa.name/projects/git-crypt/) is similar to Blackbox and more slick, but no updates in the last two years.

Of course, an option could be to pass secrets through a back-channel, but no one really likes that :)

**Comment by cscheid** [comment]
Yes, I would prefer to avoid backchannels (even though that’s what we’ve been using.)

I don’t mind using Keybase. My main question is: do you have experience with it? You’d have 3 years to teach people to do it :)

-carlos

**Comment by codementum** [comment]
Are we set on videos? I know we're set with keybase :+1:
